examples/model.py

```diff
@@ -96,3 +105,35 @@ def forward(
         )
 
         return self.head(x_dict[entity_table][: seed_time.size(0)])
+
+    def forward_id_awareness(
+        self,
+        batch: HeteroData,
+        entity_table: NodeType,
+        dst_table: NodeType,
+    ) -> Tensor:
+        if self.id_awareness_emb is None:
+            raise RuntimeError(
+                "id_awareness must be set True to use forward_id_awareness"
+            )
+        seed_time = batch[entity_table].seed_time
+        x_dict = self.encoder(batch.tf_dict)
+        # Add ID-awareness to the root node
+        x_dict[entity_table][: seed_time.size(0)] += self.id_awareness_emb.weight
```

Comment:
This doesn't do anything for 2-hop GNNs? Do we need it?

Reply:
No it does not do anything, but I still think it's good to add.